core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -196,6 +201,31 @@ public boolean isServing() {
     return mIsServing;
   }
 
+  @Override
+  public List<ConfigInfo> getConfigInfoList() {
+    List<ConfigInfo> configInfoList = new ArrayList<>();
+    String alluxioConfPrefix = "alluxio";
+    Set<String> alluxioConfExcludes = Sets.newHashSet(
+        PropertyKey.MASTER_WHITELIST.toString());
```

Comment:
It doesn't look like MASTER_WHITELIST is related to configuration whitelisting

```java
  public static final PropertyKey MASTER_WHITELIST =
      new Builder(Name.MASTER_WHITELIST)
          .setDefaultValue("/")
          .setDescription("A comma-separated list of prefixes of the paths which are "
              + "cacheable, separated by semi-colons. Alluxio will try to cache the cacheable "
              + "file when it is read for the first time.")
          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
          .setScope(Scope.MASTER)
          .build();
```

Reply:
MASTER_WHITELIST is not related to configuration, so here we do not include and show.
Only include the configuration properties that if (key.startsWith(alluxioConfPrefix) && !alluxioConfExcludes.contains(key)) 

In alluxio web, we show configuration properties (except alluxio.master.whitelist) in the upper part of the web configuration page and then show alluxio.master.whitelist info in the rest of the page. 

Should I also show the alluxio.master.whitelist info in report configuration just like what web does(other property values in the top, whitelist in the bottom)?

Reply:
I see, the UI had special handling so it didn't include it under the normal configuration view. For the CLI I think we can handle that property the same as all other property keys. I'd rather keep them alphabetical instead of moving one special property to the end.